_workinggroups/pyhep.md

```diff
@@ -45,15 +45,38 @@ The **PyHEP workshops** are a series of workshops initiated by Eduardo Rodrigues
 with the aim to provide an environment to discuss and promote the usage of Python in the HEP community at large.
 
 {:.table .table-hover .table-condensed .table-striped}
-| *Workshop* | *Location*             | *Date*               | *Agenda link*                                                                 |
-| ---------- | ---------------------- | ---------------------| ----------------------------------------------------------------------------- |
-| PyHEP 2021 | Virtual workshop       | July 5-9, 2021       | [Indico](https://indico.cern.ch/e/PyHEP2021){:target="_pyhep2021_indico"}     |
-| PyHEP 2020 | Virtual workshop       | July 13-17, 2020     | [Indico](https://indico.cern.ch/e/PyHEP2020){:target="_pyhep2020_indico"}     |
-| PyHEP 2019 | Abingdon, U.K.         | October 16-18, 2019  | [Indico](https://indico.cern.ch/e/PyHEP2019){:target="_pyhep2019_indico"}     |
-| PyHEP 2018 | Sofia, Bulgaria        | July 7-8, 2018       | [Indico](https://indico.cern.ch/event/694818/){:target="_pyhep2018_indico"}   |
+| *Workshop* | *Location*             | *Date*                | *Agenda link*                                                                 |
+| ---------- | ---------------------- | ----------------------| ----------------------------------------------------------------------------- |
+| PyHEP 2021 | Virtual workshop       | September 12-16, 2022 | [Indico](https://indico.cern.ch/e/PyHEP2022){:target="_pyhep2022_indico"}     |
```

Comment:
Should be `PyHEP 2022`

Reply:
Oh, and I did check ... I thought! Thanks, fixed!